content/blog/announcing-kudo-0.6.0.md

```diff
@@ -0,0 +1,53 @@
+---
+date: 2019-09-09
+---
+
+# Announcing KUDO 0.6.0
+
+We are proud to announce the release of KUDO v0.6.0! 
```

Comment:
In order to get more value out of the excerpts on https://kudo.dev/blog/, it would be great to summarize the new features here instead of copy/pasting the _we are proud to …_ as we did in the past.

I'm happy to suggest something!